devdocs/macos-developer-id.md

```diff
@@ -0,0 +1,12 @@
+## Expired Apple Developer ID
+
```

Comment:
Can you please explain that a symptom of the certificate needing to be updated is an error in the macOS upload jobs on `master` looking something like
```
Codesigning file dmg/Julia-1.12.app/Contents/MacOS/applet with identity 2053E9292809B66582CA9F042B470C0929340362
error: The specified item could not be found in the keychain.
```
?  That was very unclear to me when I saw it the other day.